src/app/core/form-dialog/row-details/row-details.component.ts

```diff
@@ -69,26 +69,33 @@ export class RowDetailsComponent {
     @Inject(MAT_DIALOG_DATA) public data: DetailsComponentData,
     private formService: EntityFormService,
   ) {
-    this.init(data);
+    this.init(data)
+      .then()
+      .catch((reason) => console.log(reason)); // todo this should not called in constructor?
```

Comment:
yes, let's move to ngOnInit?

what are possible causes for this to throw?

Reply:
moved to ngOnInit, lets double check if everything works as expected... tests are green